viewing_party/party.py

```diff
@@ -1,23 +1,276 @@
 # ------------- WAVE 1 --------------------
+from collections import Counter

```

Comment:
These imports I'm guessing were auto-imported by VS Code and shouldn't be here. I recommend turning off that feature in the VS Code Settings.